include/openssl/core.h

```diff
@@ -25,6 +25,11 @@ extern "C" {
  * to communicate data between them.
  */
 
+/* Opaque handles used by functions from the core */
```

Comment:
Hmmm...  "Opaque handles to be used with core upcall functions"

Reply:
Done